subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/viewing_debugging_dependencies.adoc

```diff
@@ -88,6 +88,62 @@ Given a dependency in the dependency graph you can identify the selection reason
 include::{samplesPath}/dependencyManagement/inspectingDependencies/dependencyInsightReport/dependencyInsightReport.out[]
 ----
 
+=== Understanding selection reasons
+
+The "Selecion reasons" part of the dependency insight report will list the different reasons that a dependency was selected.
+Have a look at the table below to understand the meaning of the different terms used:
+
+.Selections reasons terminology
+[%header%autowidth,compact]
+|===
+| Reason    | Meaning
+
+| (Absent)
+| This means that no other reason than having a reference, direct or transitive, was present
+
+| Was requested : <text>
+| The dependency appears in the graph, and the inclusion came with a <<declaring_dependencies#sec:documenting-dependencies, `because` text>>.
+
+| Was requested : didn't match versions <versions>
+| The dependency appears in the graph, with a <<dynamic_versions#sub:declaring_dependency_with_dynamic_version, dynamic version>>, which did not include the listed versions.
+This can also be followed by a `because` text.
+
+| Was requested : reject version <versions>
+| The dependency appears in the graph, with a <<rich_versions#, rich version>> containing one or more `reject`.
+This can also be followed by a `because` text.
+
+| By conflict resolution : between versions <version>
+| The dependency appeared multiple times in the graph, with different version requests.
+This resulted in <<dependency_resolution#sec:version-conflict, conflict resolution>> to select the most appropriate version.
+
+| By constraint
+| A <<dependency_constraints#sec:adding-constraints-transitive-deps, dependency constraint>> participated in the version selection
+
+| By constraint : <text>
+| Same as above, with the addition of a `because` text
+
+| By ancestor
+| There is a <<rich_versions#, rich version>> with a `strictly` in the graph which enforces the version of this dependency.
+
+| Selected by rule
+| A <<resolution_rules#, dependency resolution rule>> overruled the default selection process
+
+| Selected by rule : <text>
+| Same as above, with the addition of a `because` text
+
```

Comment:
Same here. I start to wonder if we should mention it once above/below the table that most reason may come with an optional `because` text

Reply:
Well since not _all_ can have a `because` text, let's leave it explicit.

Reply:
Though I'll collapse the two rows.